client/my-sites/site-settings/protect.jsx

```diff
@@ -137,31 +137,31 @@ class Protect extends Component {
 
 							{ ipAddress && (
 								<Button
-									className="protect__add-to-whitelist site-settings__add-to-whitelist"
-									onClick={ this.handleAddToWhitelist }
-									disabled={ disabled || isIpWhitelisted }
+									className="site-settings__add-to-explicitly-allowed-list"
+									onClick={ this.handleAddToAllowedList }
+									disabled={ disabled || isIpAllowed }
 									compact
 								>
-									{ isIpWhitelisted
-										? translate( 'Already in whitelist' )
-										: translate( 'Add to whitelist' ) }
+									{ isIpAllowed
+										? translate( 'Already in list of allowed IPs' )
+										: translate( 'Add to list of allowed IPs' ) }
 								</Button>
 							) }
 						</p>
 
 						<FormLabel htmlFor="jetpack_protect_global_whitelist">
-							{ translate( 'Whitelisted IP addresses' ) }
+							{ translate( 'Allowed IP addresses' ) }
 						</FormLabel>
 						<FormTextarea
 							id="jetpack_protect_global_whitelist"
-							value={ this.getProtectWhitelist() }
+							value={ this.getProtectAllowedIps() }
 							onChange={ onChangeField( 'jetpack_protect_global_whitelist' ) }
 							disabled={ disabled }
 							placeholder={ translate( 'Example: 12.12.12.1-12.12.12.100' ) }
 						/>
 						<FormSettingExplanation>
 							{ translate(
-								'You may whitelist an IP address or series of addresses preventing them from ' +
+								'You may explicitly allow an IP address or series of addresses preventing them from ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **28** times:
`translate( 'You may whitelist an IP address or series of addresses preventing them from ever being blocked by Jetpack. IPv4 and IPv6 are acceptable. To specify a range, enter the low value and high value separated by a dash. Example: 12.12.12.1-12.12.12.100' )` *ES Score: 10*